vuepress/docs/next/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -4,138 +4,82 @@ sidebarDepth: 2
 
-The bundle can be published using the CLI or the steps can be performed manually.
-
-### CLI Steps
-1. Change to the project directory if needed
-```sh
-cd example-bundle
-```
-
-2. Initialize the Entando project and accept the defaults
-``` sh
-ent prj init
-```
-
-3. Initialize the publication system. This step requires the empty Git repository URL (ending in .git) and your Git credentials.
-``` sh
-ent prj pbs-init
-```
-
-4. Publish the bundle to Git. By convention, the first version is assigned the tag `v0.0.1`, but the prefix "v" is optional.
-``` sh
-ent prj pbs-publish
-```
-Running just the command `ent prj pbs-publish` will quickly push subsequent iterations of the bundle to Git. You will be asked to input the bundle version each time. You must be consistent with versioning format and alphanumeric precedence to ensure that iterations are listed in the correct order. 
-
-5. The bundle can now be deployed into the Entando Component Repository with one command
-``` sh
-ent prj deploy
-```
- The `prj deploy` command uses the Git repository URL and project name (e.g. `example-bundle`) to create the custom resource.
-
-Jump to [Install the Bundle into an Application](#install-the-bundle-into-an-application) to finish installing your bundle.
-
-### Manual Steps 
-1. Change to the bundle directory if needed
-``` sh
-cd example-bundle/bundle
-```
-
-2. Run the following commands to initialize Git and commit the files
-``` sh
-git init
-git add .
-git commit -m "Init Git repository"
-```
-
-3. Add your remote repository as origin and push the bundle
-``` sh
-git remote add origin https://your/remote/repository.git
-git push -u origin master
-```
-
-4. Publish a Git tag
-``` sh
-git tag -a "v0.0.1" -m "My first tag"
-git push --tags
-```
-
-5. Now that you've published your bundle to Git, you can create the Kubernetes custom resource for it.
-
- Run the `ent bundler from-git` command, providing your remote Git repository URL via the `--repository` option and the correct namespace via `--namespace`. You can also include a thumbnail for your bundle with `--thumbnail-file` or `--thumbnail-url`.
-
-``` sh
-ent bundler from-git --name=example-bundle --namespace=entando --repository=https://your/remote/repository.git --dry-run > example-bundle.yaml
-```
-
-6. Apply this definition to Kubernetes. You may need to first transfer the file to your VM, e.g with `multipass transfer`.
+1. From the root bundle directory, generate the Docker image(s):
+   ``` sh
+   ent bundle pack
+   ```
+   The `pack` command builds the bundle components, then constructs a Docker image for each microservice and one for the bundle itself.
+
+2. Publish the Docker image(s) to a Docker registry:
+   ``` sh
+   ent bundle publish
+   ```
+   When prompted, input the organization of your choice, e.g. the user ID of your Docker account.
+
+3. Deploy the bundle to the ECR of your Entando instance. Specify the Docker organization where the widget is published and the bundle name.
```

Comment:
Change to "Deploy the bundle to your Entando Application using the bundle Docker URL."